src-tauri/src/algorithms/stoer_wagner.rs

```diff
@@ -49,6 +51,29 @@ pub fn stoer_wagner(graph: &mut StoerWagnerGraph) -> Cut {
     }
 }
 
+pub fn recover_mincut(
```

Comment:
How does this differ from #80?

Reply:
Off for some reason the changes from previous branch leaked here? They don't differ.